nada_numpy/array.py

```diff
@@ -948,6 +948,31 @@ def test_shuffle_randomness(vector_size, num_shuffles):
 
         return arr
 
+    def var(self) -> "NadaArray":
+        """
+        Compute the variable of array elements.
+
+        Returns:
+            nadaarray: the variance of array elements
+        """
+
+        # Compute the mean
+        m = self.mean()
+
+        # Initialise the sum to 0
+        sum = Integer(0)
+
+        # compute (x_i - mean)^2
+
+        for v in self.inner:
```

Comment:
You should profit from broadcasting here. The function as it is, may not be working as you expect.